Tools/ardupilotwaf/boards.py

```diff
@@ -223,6 +223,10 @@ def configure_env(self, cfg, env):
         ]
         env.LIB += cfg.env.LIB_RT
 
+        cfg.check_cfg(package='libsystemd', mandatory=False, global_define=True,
+                      args = ['--libs', '--cflags'])
```

Comment:
I would prefer this check to have a function similar to _check_librt_, this way it can be reused if ever needed.